### ISSUE TITLE
Router description states the wrong number of output directions.

### ISSUE THREAD

**Issue by spaghetus** [Anuken/Mindustry#4278]
**Platform**: Linux

**Build**: Steam build 122.1

**Issue**: Routers are described in-game and on the Wiki as being able to route items in three directions, but they are actually able to route backwards, which is a fourth direction. This could confuse new players, since this behaviour isn't obvious.

**Steps to reproduce**:

* View the Router's description in-game or on the Wiki

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*
- [x] **I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.**
- [x] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by itcannotbe** [comment]
This could have beem a PR but I digress.

Also, saying 3 rather than 4 can be better understood by a newer player whom the block info information is targeted to.

**Comment by BlueWolf3682** [comment]
one for input... are you saying routers have 5 total input/output sides

**Comment by spaghetus** [comment]
I was just about to file a pull request.

@BlueWolf3682 If a block that can output and input items from the same side (like a Junction or an Overflow Gate) is next to a router, it's possible for the router to send an item back into it.

**Comment by BlueWolf3682** [comment]
routers shouldnt be able to backflow into overflow gates

**Comment by spaghetus** [comment]
I just tried and it looks like it doesn't work directly, but it does if there's a junction between the router and the gate

**Comment by claw4oo** [comment]
Maybe rewording it to something along the lines of "A router inputs in four directions, and outputs in four directions, with an item buffer of 1"

**Comment by itcannotbe** [comment]
Maybe it routes it's inputs in the directions it can output to.

**Comment by kazhmir** [comment]
How about:
"A router can receive input from any side and output to any side. It has a internal capacity of 1."

I think this rule shows up when you create a n*n block of routers and it basically becomes a n*n router itself, is there an example of this rule in the wiki?

**Comment by DeltaNedas** [comment]
you dont need to know java to edit a single character in a bundle

**Comment by Anuken** [comment]
> but they are actually able to route backwards, which is a fourth direction

Not in any normal circumstances. If you know enough about them to use backflow, then you don't need the description anyway.
